addon/components/polaris-form.js

```diff
@@ -0,0 +1,132 @@
+import Component from '@ember/component';
+import layout from '../templates/components/polaris-form';
+import { normalizeAutoCompleteProperty } from '../utils/normalize-auto-complete';
+
+export default Component.extend({
+  tagName: 'form',
+  attributeBindings: [
+    'acceptCharset',
```

Comment:
Should this be `accept-charset`?

Reply:
it will be, Ember will dasherize camelCase in attribute /class name bindings